### ISSUE TITLE
Question Regarding ScreenRecording.

### ISSUE THREAD

**Issue by latortuga71** [NaejEL/flipperzero-i2ctools#3]
Hello! thanks for all your work on this repo, its helped me alot with my own flipper zero apps. I was wondering how you were able to record the screen on the flipper and convert it to a GIF to be used in the readme?

**Comment by NaejEL** [comment]
Hi,
Glad it helps you doing your own app. Since they're no documentation yet I've looked at other apps too. I actualy looking at "Scenes" instead of "Viewport" for menus but it's hard to found ressource on how to use it properly. They're an old post on the [flipper forum](https://forum.flipperzero.one/t/how-to-develop-a-3rd-party-module-plugin/2076/5) talking about it.

For screen recording I used [qFLipper](https://github.com/flipperdevices/qFlipper) to capture flipper screen and buttons, recorded with SimpleSreenRecorder and tehn converted to gif with ffmpeg.
